presto-main/src/main/java/com/facebook/presto/operator/aggregation/MaxAggregation.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation;
+
+import com.facebook.presto.metadata.OperatorType;
+import com.facebook.presto.metadata.Signature;
+import com.google.common.collect.ImmutableList;
+
+import static com.facebook.presto.metadata.Signature.orderableTypeParameter;
+
+public class MaxAggregation
+        extends AbstractMinMaxAggregation
+{
+    public static final MaxAggregation MAX_AGGREGATION = new MaxAggregation();
+    private static final OperatorType operatorType = OperatorType.GREATER_THAN;
+    private static final String NAME = "max";
+    private static final Signature SIGNATURE = new Signature(NAME, ImmutableList.of(orderableTypeParameter("E")), "E", ImmutableList.of("E"), false, false);
+
+    @Override
+    public String getName()
+    {
+        return NAME;
+    }
+
+    @Override
+    public OperatorType getOperatorType()
+    {
+        return operatorType;
+    }
+
+    @Override
+    public Signature getSignature()
```

Comment:
I'd make SIGNATURE a member variable and create it in the constructor. Also, on that note I'd remove getName() and getOperatorType and add a constructor that takes them as arguments in the base class